test/host/wasi_nn/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+wasmedge_add_executable(wasiNNTests
+  wasi_nn.cpp
+)
+
+if(WASMEDGE_WASINN_BUILD_OPENVINO)
+  add_definitions(-DWASMEDGE_WASINN_BUILD_OPENVINO)
+  message( STATUS "Download ML artifacts to ${CMAKE_CURRENT_BINARY_DIR}/wasinn_openvino_fixtures") 
+  execute_process(
+    COMMAND bash ${CMAKE_SOURCE_DIR}/utils/wasi-nn-test/download_openvino_fixtures.sh ${CMAKE_CURRENT_BINARY_DIR}/wasinn_openvino_fixtures
+    RESULT_VARIABLE DOWNLOAD_ERROR
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+  file(MD5 ${CMAKE_CURRENT_BINARY_DIR}/wasinn_openvino_fixtures/mobilenet.bin CHECKSUM_WEIGHT)
+  file(MD5 ${CMAKE_CURRENT_BINARY_DIR}/wasinn_openvino_fixtures/mobilenet.xml CHECKSUM_DESCRIP)
+  file(MD5 ${CMAKE_CURRENT_BINARY_DIR}/wasinn_openvino_fixtures/tensor-1x224x224x3-f32.bgr CHECKSUM_TENSOR)
+  if(NOT CHECKSUM_WEIGHT STREQUAL "ae096b1f735f1e8e54bac8b2a42303bd")
+    message( STATUS "mobilenet.bin downloaded with wrong md5") 
```

Comment:
Since the wrong input file may cause the segmentation fault when running OpenVINO. I would like to make cmake fatal here.